frontend/archive-node-blocks/create_schema.sql

```diff
@@ -0,0 +1,89 @@
+CREATE TABLE public_keys
```

Comment:
This looks suspiciously similar to the [archive node's schema](https://github.com/MinaProtocol/mina/blob/develop/src/app/archive/create_schema.sql). If they're supposed to be kept in sync, should this be a symlink instead?

Reply:
Ah, good call! I'll add a symlink for this, thanks for the catch!